src/components/Menus/Sidebar/SettingsPanel/SettingsPanel.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import SidebarPanel from "../SidebarPanel";
+import ThemeToggle from "./ThemeToggle/ThemeToggle";
+import FontSizeSelector from "./FontSizeSelector/FontSizeSelector";
+
+import "./SettingsPanel.scss";
+
+const SettingsPanel = () => {
+  const { t } = useTranslation();
+
+  return (
+    <SidebarPanel heading={t("settingsPanel.info")}>
+      <div className="settings-panel">
+        <h3>{t("sidebar.settingsMenu.theme")}</h3>
+        <ThemeToggle />
+        <h3>{t("sidebar.settingsMenu.textSize")}</h3>
+        <FontSizeSelector />
```

Comment:
🔧 I'd be tempted on making a flexbox `settings-panel__items` with the `theme` and `textSize` elements as `settings-panel__item` so the gap can be applied consistently